### ISSUE TITLE
Create a guide for building a webapp with JSF

### ISSUE THREAD

**Issue by evelinec** [OpenLiberty/guides-common#30]
Guide for building web application with JSF.

**Comment by bcpswope** [comment]
adding ID required ONLY to test viewing on ZenHub, will remove.

**Comment by yeekangc** [comment]
Reconcile this with #35 and see if we need 2 guides or it should only be 1 guide.

**Comment by gcharters** [comment]
@yeekangc @aguibert - if these two JSF-related guides cover what I think they're meant to cover then they're different.  #30 is focused on developing a Web app using JSF and #35 is focused on plugging in different implementations of the JSF container.  I therefore thing we can prioritize this guide.  Do you agree or are we waiting for #35 to be completed before we prioritise this guide?

**Comment by yeekangc** [comment]
I agree, @gcharters, if we can agree they serve different purposes and there is value in having both.  If so, we can move forward to prioritize.

**Comment by yeekangc** [comment]
Removing "question" tag based on the above discussion.

**Comment by yeekangc** [comment]
Thank you, @shin19991207 @gkwan-ibm.  Please proceed to publish.

**Comment by gkwan-ibm** [comment]
close
- the guide is published at https://openliberty.io/guides/jakarta-faces.html
